arb_os/arbsys.mini

```diff
@@ -276,7 +290,7 @@ impure func arbsys_sendTxToL1(topFrame: EvmCallFrame, calldata: ByteArray) { //
                                 uint(destAddr),
                             ),
                             96,
-                            inbox_currentBlockNumber(),
+                            inbox_currentArbBlockNumber(),
```

Comment:
Might want to also output the eth block number for this function

Reply:
Added the Eth block number, immediately after the Arbitrum block number.